src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -169,27 +169,38 @@ private void changeProject(final Project newProject, final long minusSeconds) {
       }
       if (model.remindIfNotesAreEmpty.get()) {
          final Work currentWork = model.activeWorkItem.get();
-         if (currentWork != null && currentWork.getNotes().isEmpty()) {
-            final TextInputDialog noteDialog = new TextInputDialog();
-            noteDialog.setTitle("Empty Notes");
-            noteDialog.setHeaderText("Switch projects without notes?");
-            noteDialog.setContentText(
-                  "What did you do for project '" + model.activeWorkItem.get().getProject().getName() + "' ?");
-            noteDialog.initOwner(mainStage);
-
-            final Optional<String> result = noteDialog.showAndWait();
-            if (result.isPresent()) {
-               currentWork.setNotes(result.get());
-            } else {
-               // cancel pressed
-               return;
-            }
+         if (currentWork != null && currentWork.getNotes().isEmpty() ) {
+            if (showNoNoteSelected(currentWork)) return;
+         }
+      }
+      if (model.remindIfNotesAreEmptyIsWork.get()) {
```

Comment:
the _remindIfNotesAreEmpty_ and _remindIfNotesAreEmptyIsWork_ should work connected. Both can be true. In theory the new option only makes sense if the existing one is already active (but we do not need to add logic for this).

PseudoCode:
if (remindIfNotesAreEmpty && notesAreEmpty)
    if (!remindIfNotesAreEmptyIsWork || isWork)
       promptForNote